### ISSUE TITLE
Became invisible except for cow head adornments

### ISSUE THREAD

**Issue by frawhst** [goonstation/goonstation#5097]
[MAJOR]

**Describe the bug**
After having both my legs removed and reattached, I became invisible, except for my cow snout, eyes, and horns. (and some random blood decals? dunno). This also affected the areas people could click to interact with me- you needed to click on the roughly 20 visible pixels if you wanted to do any action to me.

The following things didn't fix this:
Paint (only the visible parts were painted)
Showering
Acetone

**To Reproduce**
1. Have both your legs cut off
2. Reattach them
3. invisicow?

**Expected behavior**
Sprite exists and is not invisible

**Screenshots**
![invisicow](https://user-images.githubusercontent.com/67879783/121480113-94d5c800-c97f-11eb-9294-4579d15a3875.png)

**Additional context**
I was a vampire.

**Comment by eX-n0x** [comment]
I did a quick test and can confirm this happens. Tried it on a local server with some dummies transformed into various mutant races. Only seemed to have happened when either both arms or legs were removed and it only happened with cows, roaches, and lizards all without having to be antags. Debug overlay on them did show a bunch of sprite parts were missing and while unsure if this is relevant, interacting with their severed limbs through the right click menu ignores picking up the limb and attempts to interact with any nearby mobs if they happened to be right next to you in one of the four cardinal directions.
